app/src/main/java/com/moonpig/mvisample/productdetail/ProductDetailActivity.kt

```diff
@@ -42,14 +45,16 @@ class ProductDetailActivity : BaseActivity(), ProductDetailView {
                 .addToDisposables()
 
         viewModel.bindIntents(Observable.mergeArray(
-            initialIntent()
+                initialIntent()
```

Comment:
Is the indenting right here?

Reply:
Yes, it was originally wrong. Our continuation indent is 8 spaces